lib/models/data_controller.dart

```diff
@@ -9,7 +9,7 @@ final getDataFuture =
 
 class GetDataFromApi extends ChangeNotifier {
```

Comment:
no need to use the change notifier when get data from api , just wait for it in the provider